src/ts/components/extensions/carousel/Carousel.tsx

```diff
@@ -3,12 +3,14 @@ import { MantineSpacing, StyleProp } from "@mantine/core";
 import { BoxProps } from "props/box";
 import { DashBaseProps } from "props/dash";
 import { StylesApiProps } from "props/styles";
-import React from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
did you use `useState`?

Reply:
> did you use useState?


...only while messing around with thing that didn't work haha
Need to add linting and formatting to this project  in order to clean up stuff like this automatically,